config/config_test.go

```diff
@@ -54,7 +54,13 @@ func (s *ConfigSuite) TestUnmarshal(c *C) {
 [init]
 		defaultBranch = main
 [url "ssh://git@github.com/"]
-	insteadOf = https://github.com/
+		insteadOf = https://github.com/
+[credential]
+		helper = osxkeychain
+		helper = "!f() { echo 'password=multiple-helpers'; }; f"
+		helper = "/bin/get-my-git-password"
+[credential "https://kernel.org"]
+		username = foo
```

Comment:
Say the config file did not have a default entry i.e. only this
```
[credential "https://kernel.org"]
		username = foo
```

What would be populated for `Credentials["default"]`? Should there be a default entry in this case?